src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -771,19 +642,20 @@ private int getNumberOfHits(String page, String marker, Pattern pattern) throws
      */
     private String getResults(URL source) throws IOException {
 
-        InputStream in = source.openStream();
-        StringBuilder sb = new StringBuilder();
-        byte[] buffer = new byte[256];
-        while (true) {
-            int bytesRead = in.read(buffer);
-            if (bytesRead == -1) {
-                break;
-            }
-            for (int i = 0; i < bytesRead; i++) {
-                sb.append((char) buffer[i]);
+        try(InputStream in = source.openStream()) {
+            StringBuilder sb = new StringBuilder();
```

Comment:
Yes, that is the code I'm unifying. The block below reads from a file though (although it should be similar, I've kept it for now). Identical code in ACMFetcher and almost identical in Util.GetResultsWithEncoding. Will push in a few minutes.